tests/test_staff.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+import nose
+
+from models.staff import Staff
+
+class TestStaff(unittest.TestCase):
+	"""Test cases for the Staff class"""
+
+	def setUp(self):
+		"""Setup a new Staff object to be used for testing"""
+		self.staff = Staff('Percila','Njira')
+
+	def test_staff_instance(self):
+		"""Tests if the object created is an instance of the Staff class"""
+		self.assertIsInstance(self.staff,Staff)
+
+	def test_staff_type(self):
+		"""Tests if the object created is of type Staff"""
+		self.assertEqual(type(self.staff),Staff)
+
+	def test_person_type(self):
+		"""Tests if the Person type is Staff """
+		self.assertEqual(self.staff.person_type,'STAFF')
+
+	def test_person_name(self):
+		"""Tests if the full name is a combination of the first name and last name"""
+		self.assertEqual(self.staff.name,'PERCILA NJIRA')
+
+	def test_person_office_allocated(self):
+		"""Tests if at creation, a new staff has no office allocated"""
+		self.assertEqual(self.staff.office_allocated,False)
+
+	def test_person_office(self):
+		"""Tests if the Staff created has no office"""
+		self.assertEqual(self.staff.office,'')
+
+	def test_person_livingspace_allocated(self):
+		"""Tests if at creation, a new staff has no Living Space allocated"""
+		self.assertEqual(self.staff.livingspace_allocated,False)
+
+	def test_person_livingspace(self):
+		"""Tests if the Staff created has no Living Space"""
+		self.assertEqual(self.staff.livingspace,'')
+
+	def test_check_staff_id(self):
+		"""Check if the id assigned to a fellow is unique to only Staff members"""
+		first_char = self.staff.staff_id[0]
+		self.assertEqual(first_char,'S')
```

Comment:
Please review this method's  docstring. 
You could create a number of Staff objects and compare their `staff_id`s. I think that would make this test more conclusive.